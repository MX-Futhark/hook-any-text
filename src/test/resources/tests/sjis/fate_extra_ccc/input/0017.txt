816381638CBE82C182BD81420A82A282DC81418A6D82A982C992668CBE82B582BD81420000000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F0000001000000B0000000000000000000000000000000000000000001F01000000000000000000000000FFFFFFFF0000394300003C42000000000000604100000000FFFFFFFF0000474300003C42000000000000000000006041FFFFFFFF0000394300007442000000000000604100006041FFFFFFFF00004743000074420000000030F3A248020000006014C908000000000000000000000000FFFFFFFF00000000F8F1A248020000008014C908000000000000000000000000FFFFFFFF0001000060F2A24802000000A014C908000000000000000000000000FFFFFFFF00010000C8F2A24802000000C014C908000000000000000000000000FFFFFFFF0001000030F3A2480200000000000000000000000000000000000000FFFFFFFF0001000098F3A248020000000015C908000000000000000000000000030001000000000000F4A248020000002015C908000000000000000000000000030001000000000068F4A248020000004015C9080000000000000000000000000300010000000000D0F4A248020000006015C908000000000000000000000000030001000000000038F5A248020000008015C9080000000000000000000000000300010000000000A0F5A24802000000A015C908000000000000000000000000030001000000000008F6A24802000000C015C908000000000000000000000000030001000000000070F6A24802000000E015C9080000000000000000000000000300010000000000D8F6A248020000000016C908000000000000000000000000030001000000000040F7A248020000002016C9080000000000000000000000000300010000000000A8F7A248020000004016C90800