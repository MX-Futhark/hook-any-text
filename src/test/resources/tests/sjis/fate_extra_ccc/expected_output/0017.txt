["……言った。
いま、確かに断言した。"]